code/modules/arcade/arcade_prize.dm

```diff
@@ -17,8 +22,68 @@
 	playsound(src.loc, 'sound/items/bubblewrap.ogg', 30, 1, extrarange = -4, falloff = 10)
 	icon_state = "prizeconfetti"
 	src.color = pick(random_color_list)
-	var/prize_inside = pick(/obj/random/carp_plushie, /obj/random/plushie, /obj/random/figure, /obj/item/toy/eight_ball)	//will add ticket bundles later
+	var/prize_inside = pick(possible_contents)
 	spawn(10)
 		user.unEquip(src)
-		new prize_inside(user.loc)
-		qdel(src)
\ No newline at end of file
+		if(istype(prize_inside, /obj/item/stack))
+			var/amount = pick(5, 10, 15, 25, 50)
+			new prize_inside(user.loc, amount)
+		else
+			new prize_inside(user.loc)
+		qdel(src)
+
+/obj/item/toy/prizeball/mech
+	name = "mecha figure capsule"
+	desc = "Contains one collectible mecha figure!"
+	possible_contents = list(/obj/random/mech)
+
+/obj/item/toy/prizeball/carp_plushie
+	name = "carp plushie capsule"
+	desc = "Contains one space carp plushie!"
+	possible_contents = list(/obj/random/carp_plushie)
+
+/obj/item/toy/prizeball/plushie
+	name = "animal plushie capsule"
+	desc = "Contains one cuddly animal plushie!"
+	possible_contents = list(/obj/random/plushie)
+
+/obj/item/toy/prizeball/figure
+	name = "action figure capsule"
+	desc = "Contains one action figure!"
+	possible_contents = list(/obj/random/figure)
+
+/obj/item/toy/prizeball/therapy
+	name = "therapy doll capsule"
+	desc = "Contains one squishy therapy doll."
+	possible_contents = list(/obj/random/therapy)
+
+/obj/item/stack/tickets
+	name = "prize ticket"
+	desc = "Prize tickets from the arcade. Exchange them for fabulous prizes!"
+	singular_name = "prize ticket"
+	icon = 'icons/obj/arcade.dmi'
+	icon_state = "tickets_1"
+	force = 1
```

Comment:
should probably be 0; it doesn't hurt to get hit by these IRL, that's for sure, and a lot heavier things have a force of 0 than this (ie: books).